### PR TITLE
Fetch event to display

### PR DESCRIPTION
**Pull request by khanguyen1207**

### CODE REVIEW

**Comment by shamnadps** [comment]
Is this ready for review?

**Comment by khanguyen1207** [comment]
Yes, Shamnad.

File: frontend/src/views/consumer/paymentstatus-modal/index.js [hunk 1]
```diff
@@ -70,11 +78,13 @@ export default withTheme(
             path="/consumer/payment"
             children={({ match, location }) => {
               const query = parseQuery(location.search);
+              if (!this.state.eventToLoad) {
+                this.loadEvent(query.event_id);
+              }
               const meaning = match && this.getMeaning(query.status);
               const longText = match && this.getMeaningLong(query.status);
               const foundEvent = match && store.findEvent(query.event_id); // maybe undefined if not found
```
Comment (shamnadps) [inline frontend/src/views/consumer/paymentstatus-modal/index.js]: `const foundEvent = match && store.findEvent(query.event_id); // maybe undefined if not found
const event = foundEvent && foundEvent.toJSON();`

Isn't this supposed to get the event from store based on eventId?. Then why would we need the additional loadEvent call?. Did you debug it?.

Comment (khanguyen1207) [reply frontend/src/views/consumer/paymentstatus-modal/index.js]: Without `loadEvent()` `foundEvent` would be `undefined`. I tested it with @vinujosef

**Comment by shamnadps** [comment]
@khanguyen1207 should this be merged and deployed to production after testing?

**Pull request merged**